.clippy.toml

```diff
@@ -1 +1,4 @@
 allow-mixed-uninlined-format-args = false
+disallowed-types = [
+    { path = "tower::util::BoxCloneService", reason = "Use our internal BoxCloneService which is Sync" },
```

Comment:
This reminds me, there is now https://docs.rs/tower/latest/tower/util/struct.BoxCloneSyncService.html that we may want to replace our own with.